modules/custom/openy_map/config/install/openy_map.settings.yml

```diff
@@ -6,6 +6,10 @@ leaflet:
   base_layer_override:
     enable: 0
     override: ''
+  clustering:
```

Comment:
We need to add hook_update_N for supporting upgrade path

Reply:
@podarok thank you for quick review.
Like this? Did I get it right? 
```
/**
 * Add Leaflet base layer override configuration.
 */
function openy_map_update_8004() {
  $config_updater = \Drupal::service('openy_upgrade_tool.param_updater');
  $config_dir = drupal_get_path('module', 'openy_map') . '/config/install/';
  $config_name = 'openy_map.settings';
  $config_path = $config_dir . $config_name . '.yml';
  $config_updater->update($config_path, $config_name, 'leaflet.base_layer_override');
}
```

Reply:
@podarok 
I hope i got it right. So I've added hook_update_n in my new commit https://github.com/ymcatwincities/openy/pull/2316/commits/9a80337c1221dbd3faee12396107ca9b627428db